docs/projects/slopi-communication.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Searchable Linkable Open Public Indexed (SLOPI) Communication
+nav_order: 3
+parent: Projects
+---
+
+# Searchable Linkable Open Public Indexed (SLOPI) Communication
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## What is it?
+
+Messages written in the SLOPI (pronounced "sloppy") communication style are:
+
+- Searchable: Messages can be found using Google or other search engines.
+- Linkable: Messages have a permalink.
+- Open: Messages are in the open.
+- Public: Messages are public.
+- Indexed: Messages are indexed by search engines.
+
+This phrasing, especially with regard to messages being searchable and linkable, is inspired by the [discussion][] criteria of [Core Infrastructure Initiative (CII) Best Practices][] program.
+
+Examples of communication tools that support the SLOPI style include:
+
+- mailing lists with public archives
+- public forums
+- public issue trackers
+- Gitter
+- IRC channels with public logs
+- XMPP rooms with public logs
+- public roadmaps
+- public kanban boards
+
```

Comment:
This is a great list!

I'm imagining myself as a developer, coming to this project (and maybe I use slack) and realizing "oh gosh, I don't have transparent communication!" I think what would be good to add here is links or suggestions to actual resources that are "SLOPI approved." Otherwise, the page is telling me I'm not doing it right (but not providing any help to change that.) 

And what is the incentive for being SLOPI? A badge? Being added to a list? Something else?

Reply:
I'm a little torn on recommending specific solutions. Gitter to me is a special case because it's so unique and I could add a link for Gitter if you think that would help. I'm also torn on if the links should be to software that you run yourself like mailman or if the links should go to services. I'm somewhat reluctant to link to services running proprietary software such a GitHub or Google Groups. For me, adding links and making recommendations is part of the project plan below. Could we consider the links out of scope for now and add them to a future pull request after we talk it out some more?

The incentive for being SLOPI is that your project will grow. New users and contributors will find your project more easily because more content about your project is being indexed by search engines. Users with similar concerns can more easily find and help each other.

I could cook up a badge if you'd like. I think I showed you the DOI badge I made. It's easy.

Reply:
What about the other way I suggested, going to the providers and evaluating them for being SLOPI, and then suggesting to users based on that?